mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3561,6 +3564,13 @@ async fn process_maker_reserved(ctx: MmArc, from_pubkey: H256Json, reserved_msg:
                     connected: None,
                     last_updated: now_ms(),
                 };
+
+                ctx.event_stream_manager
+                    .send_fn(OrderStatusStreamer::derive_streamer_id(), || {
```

Comment:
I don't get why we use `send_fn` and not `send` here, can you please clarify?

Reply:
If the streamer is not enabled. this code is still executed.
`send_fn` is used to lazily execute the code only until we know there is an actual streamer hooked up. so if no streamer is running, the `data_fn` closure won't get executed (the redundant `clone`ing, or any code per se).

Reply:
Thanks for the clarification, great approach :)